test/Processors/TaggedHooksFirstStrategyTests.cs

```diff
@@ -56,42 +56,6 @@ IHookMethod Create(string name, int aggregation = 0, params string[] tags)
             };
         }
 
-        //[AfterScenario("Foo")]
```

Comment:
This bit of code block was to indicate what is the logic being tested. It was to ease  the data setup. Perhaps you can add a line describing such, but i prefer we leave this in place.